guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
```

Comment:
```suggestion
= Viewing multi-environment hosts
```

Reply:
I'm going to keep it spelled out in the titles and then mention it in the first instance.